server/app/services/queryHandler/Extractor.java

```diff
@@ -26,12 +27,9 @@ public static String extractString(JsonNode json, String key) {
     } else {
 
       if (key == "description") {
-        System.out.println("Check point 1");
-        System.out.println(json.get("fields").get(key).toString());
-        return json.get("fields").get(key).textValue();
+        String description = json.get("fields").get(key).textValue();
+        return Utils.hyperlinkTicketNo(description);
```

Comment:
In Java, it is normal practice to have a single return from a method, e.g. assign what you are returning here and on line 33 to a variable and then return it at the end.